addons/resource/models/resource.py

```diff
@@ -142,9 +142,9 @@ def _merge(self, other, difference):
 def sum_intervals(intervals):
     """ Sum the intervals duration (unit : hour)"""
     return sum(
-        (stop - start).total_seconds() / 3600
+        (stop - start).total_seconds()
         for start, stop, meta in intervals
-    )
+    ) / 3600
```

Comment:
What is the purpose of this change?

Reply:
performance?

Reply:
I know 16.0 is not really stable yet, but we should still keep changes to a minimum. Besides, this change wouldn't have a meaningful effect on performance